docs/swagger.yaml

```
@@ -418,189 +610,422 @@ paths:
+                    properties:
+                      transactions:
+                        type: array
+                        items:
+                          type: string
+                          format: uuid
+                      requests:
+                        type: array
+                        items:
+                          type: string
+                          format: uuid
+                      users:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/TrapID"
+        "404":
+          description: Not Found
+    delete:
+      description: グループを削除する
+      tags:
+        - Groups
+        - Admins
       responses:
         "200":
-          description: OK
+          description: 削除に成功した。
+        "404":
+          description: Not Found
+  "/files":
+    post:
+      description: ファイルをアップロードする。
+      tags:
+        - Files
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                file:
+                  $ref: "#/components/schemas/File"
+                name:
+                  type: string
+                  example: "hoge.png"
+      responses:
+        "200":
+          description: 変更に成功した。
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/User"
-        "400":
-          description: リクエストに形式上の不備がある。
+                type: string
+                format: uuid
+        "404":
+          description: Not Found
+  "/files/{fileID}":
+    parameters:
+      - name: fileID
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+    get:
+      tags:
+        - Files
+      description: 指定されたファイルを返す
+      responses:
+        "200":
+          description: 該当するファイルが存在した 返す
+          content:
+            multipart/form-data:
+              schema:
+                type: object
+                properties:
+                  file:
+                    $ref: "#/components/schemas/File"
+                  name:
+                    format: string
+                    example: "hoge.png"
+        "404":
+          description: Not Found
+    delete:
+      description: 指定したidのファイルを削除する。
+      tags:
+        - Files
+        - Admins
+        - Authors
+      responses:
+        "204":
+          description: 正常に取り消すことができた。
         "403":
-          description: admin以外の人
+          description: 編集権限がない人(作成者、admin以外)による操作。
         "404":
-          description: adminを変更しようとした対象者は存在しない
-        "409":
-          description: 既に指定したadminの状態である
+          description: 該当する画像はない。
 
 components:
   schemas:
-    TypeEnum:
-      type: string
-      enum: [club, contest, event, public]
-    StateEnum:
+    StatusEnum:
       type: string
       enum: [submitted, fix_required, accepted, fully_repaid, rejected]
```

```suggestion
      enum: [submitted, fix_required, accepted, completed, rejected]
```
reimbursementだけではないので